src/gc/handletable.cpp

```diff
@@ -601,8 +602,8 @@ void HndLogSetEvent(OBJECTHANDLE handle, _UNCHECKED_OBJECTREF value)
         ADIndex appDomainIndex = HndGetHandleADIndex(handle);   
         AppDomain* pAppDomain = SystemDomain::GetAppDomainAtIndex(appDomainIndex);
```

Comment:
This is the only call which I wasn't sure if it should go on the vm or the gc side. Let me know if I should move it over

Reply:
That should go on the EE side, I think. Ideally the GC doesn't need to know about AppDomains.